reproducibility_project/src/engines/hoomd/project.py

```diff
@@ -70,10 +72,41 @@ def check_equilibration_npt(job):
 @Project.pre(lambda j: j.doc.get("nvt_finished"))
 @Project.post(lambda j: j.doc.get("nvt_eq"))
 def check_equilibration_nvt(job):
-    """Run a simulation with HOOMD-blue."""
+    """Check the equilibration of the NVT simulation."""
     job.doc.nvt_finished = check_equilibration(job, "nvt", "potential_energy")
 
 
+@Project.operation.with_directives({"executable": "$MOSDEF_PYTHON", "ngpu": 1})
+@Project.pre(lambda j: j.sp.engine == "hoomd")
+@Project.pre(lambda j: j.doc.get("nvt_eq"))
+@Project.post(lambda j: j.doc.get("post_processed"))
+def post_process(job):
+    """Run post-processing on the log files."""
+    from shutil import copy
+
```

Comment:
Just a comment that I did not know about this module. Super cool!!

Reply:
I didn't know about it either, but `recfunctions` is AWESOME!